webcams/transalpina.php

```diff
@@ -0,0 +1,13 @@
+<div class="webcam">
+    <span>Partia Constantinescu</span>
+    <div class="box">
```

Comment:
```suggestion
    <div>
```
No need for class as we're holding an image.